### PR TITLE
Extend EntryService by Delete, Publish and Unpublish actions

### PR DESCRIPTION
**Pull request by skrabacz-michal**

### CODE REVIEW

**Comment by skrabacz-michal** [comment]
please check @Khaledgarbaya

File: entry.go [hunk 1]
```diff
@@ -87,3 +89,48 @@ func (service *EntriesService) Get(spaceID, entryID string) (*Entry, error) {
...
+	req, err := service.c.newRequest(method, path, nil, nil)
+	if err != nil {
+		return err
+	}
+
+	return service.c.do(req, nil)
+}
+
+// Activate the entry, a.k.a publish
+func (service *EntriesService) Activate(spaceID string, entry *Entry) error {
```
Comment (tolgaakyuz) [inline entry.go]: How about renaming this method to `Publish`. and `Unpublish` for the next one?

Comment (skrabacz-michal) [reply entry.go]: Method naming is based on similar methods in 'content_type.go'.

Comment (tolgaakyuz) [reply entry.go]: I still highly recommend to use `Publish` and `Unpublish`. I will change the other occurrences of `Activate` and `Deactivate`.

Comment (skrabacz-michal) [reply entry.go]: ok

**Comment by Khaledgarbaya** [comment]
Thank you @skrabacz-michal for the contribution.

**Pull request merged**